### PR TITLE
Simplify exports

### PR DESCRIPTION
**Pull request by Pytal**
Export modules with a single line and add missing license headers

### CODE REVIEW

File: src/components/NcDashboardWidgetItem/index.js [hunk 1]
```diff
@@ -1,3 +1,23 @@
-import NcDashboardWidgetItem from './NcDashboardWidgetItem.vue'
+/**
+ * @copyright 2022 Christopher Ng <chrng8@gmail.com>
```
Comment (skjnldsv) [inline src/components/NcDashboardWidgetItem/index.js:2]: Initial creator of this file is @raimund-schluessler :wink:

File: src/components/NcDashboardWidget/index.js [hunk 2]
```diff
@@ -1,3 +1,23 @@
-import NcDashboardWidget from './NcDashboardWidget.vue'
+/**
+ * @copyright 2022 Christopher Ng <chrng8@gmail.com>
```
Comment (skjnldsv) [inline src/components/NcDashboardWidget/index.js:2]: Initial creator of this file is @raimund-schluessler :smile:

**Review by skjnldsv — APPROVED** [review]
Aside from my comments

**Review by ChristophWurst — APPROVED** [review]

**Review by juliushaertl — APPROVED** [review]

File: src/components/NcDashboardWidgetItem/index.js (hunk 1 above)
Comment (skjnldsv) [reply src/components/NcDashboardWidgetItem/index.js:2]:
```suggestion
 * @copyright 2022 Raimund Schlüßler <raimund.schluessler@mailbox.org>
```

File: src/components/NcDashboardWidget/index.js (hunk 2 above)
Comment (skjnldsv) [reply src/components/NcDashboardWidget/index.js:2]:
```suggestion
 * @copyright 2022 Raimund Schlüßler <raimund.schluessler@mailbox.org>
```

File: src/components/NcAppNavigationCaption/index.js [hunk 3]
```diff
@@ -1,3 +1,23 @@
-import NcAppNavigationCaption from './NcAppNavigationCaption.vue'
+/**
+ * @copyright 2022 Christopher Ng <chrng8@gmail.com>
```
Comment (skjnldsv) [inline src/components/NcAppNavigationCaption/index.js:2]:
```suggestion
 * @copyright 2022 Raimund Schlüßler <raimund.schluessler@mailbox.org>
```

File: src/components/NcDashboardWidgetItem/index.js (hunk 1 above)
Comment (skjnldsv) [inline src/components/NcDashboardWidgetItem/index.js:2]:
```suggestion
 * @copyright 2022 Raimund Schlüßler <raimund.schluessler@mailbox.org>
```

**Pull request merged**